### PR TITLE
[CARBONDATA-1117]updated configuration-parameters.md and useful-tips-on-carbondata.md files for SET/RESET

### PR DESCRIPTION
**Pull request by vandana7**

### CODE REVIEW

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/703/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/3299/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/724/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/3320/

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: docs/useful-tips-on-carbondata.md [hunk 1]
```diff
@@ -234,3 +233,8 @@ scenarios. After the completion of POC, some of the configurations impacting the
 | carbon.use.multiple.temp.dir | spark/carbonlib/carbon.properties | Data loading | Whether to use multiple YARN local directories during table data loading for disk load balance | After enabling 'carbon.use.local.dir', if this is set to true, CarbonData will use all YARN local directories during data load for disk load balance, that will improve the data load performance. Please enable this property when you encounter disk hotspot problem during data loading. |
 
 Note: If your CarbonData instance is provided only for query, you may specify the property 'spark.speculation=true' which is in conf directory of spark.
+
+
+
```
Comment (jackylk) [inline docs/useful-tips-on-carbondata.md]: remove empty line

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/3345/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/3346/

**Comment by ravipesala** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/747/

**Comment by ravipesala** [comment]
Build Success with Spark 1.6, Please check CI http://144.76.159.231:8080/job/ApacheCarbonPRBuilder/748/

File: docs/configuration-parameters.md [hunk 2]
```diff
@@ -143,5 +144,87 @@ This section provides the details of all the configurations required for CarbonD
...
+| carbon.options.sort.scope                | This property can have four possible values BATCH_SORT, LOCAL_SORT, GLOBAL_SORT and NO_SORT. If set to BATCH_SORT, the sorting scope is smaller and more index tree will be created,thus loading is faster but query maybe slower. If set to LOCAL_SORT, the sorting scope is bigger and one index tree per data node will be created, thus loading is slower but query is faster. If set to GLOBAL_SORT, the sorting scope is bigger and one index tree per task will be created, thus loading is slower but query is faster. If set to NO_SORT data will be loaded in unsorted manner. |
```
Comment (jackylk) [inline docs/configuration-parameters.md]: Description for GLOBAL_SORT is wrong, not per task

Comment (vandana7) [inline docs/configuration-parameters.md]: Hi jackylk
I found this description of global_sort from carbonCommonConstants Class,if it is not correct please tell me from where i can get correct description for the same.

**Comment by CarbonDataQA** [comment]
SDV Build Success with Spark 2.1, Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/250/

**Comment by chenliang613** [comment]
@vandana7  please correct PR's title as per https://github.com/apache/carbondata/blob/master/docs/How-to-contribute-to-Apache-CarbonData.md

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/31/

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**